src/settings/mod.rs

```diff
@@ -289,6 +303,10 @@ pub struct Settings {
 
     /// Settings for SMTP custom provider.
     pub smtp: Smtp,
+
+    /// Settings for SocketLabs.
+    #[serde(serialize_with = "serialize::hidden_or_not_set")]
+    pub socketlabs: Option<SocketLabs>,
```

Comment:
Didn't occur to me last night but should we add a unit test for these settings?

Reply:
Yessss!

Reply:
Done.